Dockerfile

```diff
@@ -0,0 +1,13 @@
+FROM runatlantis/atlantis:latest
+
+ENV TERRAGRUNT_VERSION=v0.21.7
```

Comment:
I would change this to `ARG`, imo it's only required during build time or?

Reply:
Good point!